snowshu/core/main.py

```diff
@@ -105,11 +110,19 @@ def create(replica_file: click.Path,
 @click.option('--barf', '-b',
               is_flag=True,
               help="outputs the source query sql to a local folder snowshu_barf_output")
-def analyze(replica_file: click.Path, barf: bool):
+@click.option(
+    '--incremental', '-i',
+    is_flag=True,
+    help="creates relations and loads data only for new entries found in replica.yml, "
+         "which are not already present in target replica image")
```

Comment:
Unless there is a specific reason to include this flag for the analyze command, we should probably exclude it, and leave `replica.incremental` as `None` or `False`.

Alternatively, if we want to keep behavior parity, this option should take an image name so that the graph difference can be calculated and only those relations end up being analyzed.

Reply:
> Unless there is a specific reason to include this flag for the analyze command, we should probably exclude it, and leave `replica.incremental` as `None` or `False`.
> 
> Alternatively, if we want to keep behavior parity, this option should take an image name so that the graph difference can be calculated and only those relations end up being analyzed.

We are taking it out for now. We can spin off the graph difference calculation in new ticket

Reply:
Yeah - I was thinking whether we should leave this here. Asked Paul to remove it